trajGen3D.py

```diff
@@ -15,14 +15,56 @@
 current_heading = np.zeros(2)
 
 def get_helix_waypoints(t, n):
-    """ The function generate n helix waypoints from the given time t
-        output waypoints shape is [n, 3]
+    """ The function generate n helix waypoints from the given time t.
+        Output waypoints shape is [n, 3]
     """
     waypoints_t = np.linspace(t, t + 2*np.pi, n)
     x = 0.5*np.cos(waypoints_t)
     y = 0.5*np.sin(waypoints_t)
     z = waypoints_t
 
+
+    return np.stack((x, y, z), axis=-1)
+
+def get_poly_waypoints(t,n):
```

Comment:
Since your get_leminiscata_waypoints does not seem to be used yet, I think we should create a folder call trajectories, and move all trajectories generation files into it and create a new one for your work and submit that change as a review.
Make sure you write some unit test in the test folder :)